graylog2-server/src/main/java/org/graylog2/rest/resources/system/ldap/LdapResource.java

```diff
@@ -298,15 +298,13 @@ public Response updateGroupMappingSettings(@ApiParam(name = "JSON body", require
             config.setCredentials(ldapSettings.getSystemPassword());
         }
 
-        try {
-            LdapNetworkConnection connection = ldapConnector.connect(config);
-            final Set<String> groups = ldapConnector.listGroups(connection,
-                                                                ldapSettings.getGroupSearchBase(),
-                                                                ldapSettings.getGroupSearchPattern(),
-                                                                ldapSettings.getGroupIdAttribute()
+        try (LdapNetworkConnection connection = ldapConnector.connect(config)) {
+            return ldapConnector.listGroups(connection,
+                                            ldapSettings.getGroupSearchBase(),
+                                            ldapSettings.getGroupSearchPattern(),
+                                            ldapSettings.getGroupIdAttribute()
             );
-            return groups;
-        } catch (LdapException e) {
+        } catch (IOException | LdapException e) {
```

Comment:
Possibly, but there are no other exceptions from those two lines, so that's probably not necessary